### PR TITLE
[ePD] Transition active PD to new CP

### PR DESCRIPTION
**Pull request by roman-karpovich**
[ch21936]

### CODE REVIEW

**Comment by robertavram** [comment]
for all active PDs, when transitioning to a new CP cycle, make sure to add the new CP Structure to the Intervention (do not remove the old one)

File: src/etools/applications/core/management/commands/init-celery.py [hunk 1]
```diff
@@ -73,4 +73,9 @@ def handle(self, *args, **options):
             'enabled': False,
             'crontab': first_day_of_the_month})
 
+        PeriodicTask.objects.get_or_create(name='Transition active PD to new CP', defaults={
```
Comment (robertavram) [inline src/etools/applications/core/management/commands/init-celery.py:76]: fixtures

File: src/etools/applications/reports/tasks.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+from django.db import connection
+from django.utils import timezone
+
+from etools.applications.partners.models import Intervention
+from etools.applications.users.models import Country
+from etools.config.celery import app
+
+
+@app.task
+def transfer_active_pds_to_new_cp():
```
Comment (robertavram) [inline src/etools/applications/reports/tasks.py:10]: move to partners

**Comment by robertavram** [comment]
needs a few updates

File: src/etools/applications/reports/tasks.py (hunk 2 above)
Comment (roman-karpovich) [reply src/etools/applications/reports/tasks.py]: updated

**Pull request merged**